styles/ranking.css

```
@@ -0,0 +1,60 @@
+.ranking {
+  padding: 20px 175px;
+  background-color: var(--bg-gray);
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-gap: 10px; /* gap BETWEEN grid items */
+}
+
+.item {
+  padding-bottom: 10px;
+}
+
+.ranking__item {
+  color: var(--text-semi-light-gray);
+  background-color: var(--bg-white);
+  font-size: 12px;
+  line-height: 18px;
+  text-align: left;
+  padding-inline: 10px;
+  position: relative;
+}
+
+.item__title {
+  font-size: 13px;
+  color: var(--text-dark-gray);
+  margin: 10px 0;
+}
+
+
+.item__image {
+  height: 100px;
+  margin: 0 auto;
+  object-fit: contain;
+}
+
+.overlay {
+  position: absolute;
+  background-color: var(--bg-main-blue);
+  top: 0%;
+  left: 0%;
+  width: 24px;
+  height: 24px;
+  transform: translate(0%, 0%);
+  display: flex;
+  align-items: center;
+  /* Additional styles for the overlay */
+}
+
+.overlay__text {
+  color: var(--text-white);
+  font-family: "Roboto";
+  font-style: italic;
+  margin: 0 auto;
+  font-size: 17px;
```

`letter-spacing`で`italic`テキストが中央に見えるというhackもあるみたいですね。先程、PR TIMES toppageから検証してみました笑
```suggestion
  font-size: 17px;
  letter-spacing: 3px;
  line-height: 25px;
```

---

なるほど, 私の書いたコード上でも上手く動くことが確認できたのでcommitします！

---

[こちらのcommit](https://github.com/ShunDeveloper/prtimes_htmlcss/pull/11/commits/bd8c52f2570ef4c6bda94df32a86e922b052b4c6)で修正しました